src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/Pkcs9MessageDigest.cs

```diff
@@ -13,16 +13,14 @@ public sealed class Pkcs9MessageDigest : Pkcs9AttributeObject
         // Constructors.
         //
 
-        public Pkcs9MessageDigest() :
-            base(Oids.MessageDigestOid.CopyOid())
+        public Pkcs9MessageDigest()
+            : base(Oids.MessageDigestOid.CopyOid())
         {
         }
 
-        internal Pkcs9MessageDigest(ReadOnlySpan<byte> signatureDigest)
+        internal Pkcs9MessageDigest(ReadOnlySpan<byte> rawData)
+            : base(Oids.MessageDigestOid.CopyOid(), rawData)
```

Comment:
As far as I could tell, this internal constructor was not used. So re-purpose it from `signatureDigest` to `rawData`.

Reply:
Yeah, looks like it was merged in dead.  Probably means that during the PR I got feedback that I could just go straight to AttributeAsn instead of creating this as an intermediate.